src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -249,6 +249,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <NativeFileReference Include="@(NativeLibrary->'%(Identity)')" Condition="'@(NativeLibrary->Count())' != '0'" />
```

Comment:
This works in typical cases, where the `NativeLibrary` is defined outside of any target in user's csproj. Which is the way how I imagine most of the people use it. It won't work if the user defines the `NativeLIbrary` inside of a target. Do we want to support such scenario? Does it make sense?

But if for